renovate-config.json

```diff
@@ -4,5 +4,21 @@
     "github>platform-engineering-org/.github",
     ":disableDependencyDashboard",
     ":pinDigestsDisabled"
```

Comment:
Please remove this setting as it already present in the top level configuration

Reply:
Adding the same preset as an upstream configuration should have no effect...so, while yours is a good suggestion, I don't think there is any hurry, and I'd prefer to wait until after we actually have things working satisfactorily.  🥴